### PR TITLE
Opsgenie release team

### PR DESCRIPTION
**Pull request by cpennington**

### CODE REVIEW

**Review by coryleeio — APPROVED** [review]
This seems fine after tests are passing as it looks like it's just linting stuff atm

File: tubular/scripts/update_recent_deployers.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+@click.option('--recent-cutoff', 'recent_cutoff', default=30, show_default=True, type=int, help="Number of minutes to consider as being recent")
+def update_recent_deployers(
+   opsgenie_api_key,
+   opsgenie_team_id,
+   gocd_pipelines,
+   gocd_username,
+   gocd_password,
+   gocd_url,
+   github_token,
+   recent_cutoff=30
```
Comment (adzuci) [inline tubular/scripts/update_recent_deployers.py]: 30++

File: tubular/scripts/update_recent_deployers.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+               raise
+
+            user = match.group('user')
+            click.echo(click.style('Removing user {!r} and retrying'.format(user), fg='red'))
+            recent_deployers.remove(user)
+            continue
+         else:
+            raise
+
+   # TODO: Make escalation to the non-deployers team happen immediately if recent-deployers is empty
```
Comment (adzuci) [inline tubular/scripts/update_recent_deployers.py]: Is DevOps the non-deployers team?

File: tubular/scripts/update_recent_deployers.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+):
+   """
+   Update an OpsGenie team to contain only those users whose changes were recently deployed
+   by a particular GoCD pipeline.
+   """
+   repo_tools_repo = GitHubAPI('edx', 'repo-tools-data', github_token)
+   people_yaml_data = repo_tools_repo.file_contents('people.yaml')
+   people_yaml = yaml.safe_load(people_yaml_data)
+
+   email_aliases = [
```
Comment (adzuci) [inline tubular/scripts/update_recent_deployers.py]: Does people.yaml have an edx.org email address for everyone?
Does anyone use two different edx emails e.g. bill vs bderusha or derf vs fred?

**Review by adzuci — APPROVED** [review]
I'm not sure how to properly test this, but I'm happy to see it merged and iterated on.

File: tubular/scripts/update_recent_deployers.py (hunk 2 above)
Comment (cpennington) [reply tubular/scripts/update_recent_deployers.py]: For edx-platform they will be, yes.

File: tubular/scripts/update_recent_deployers.py (hunk 3 above)
Comment (cpennington) [reply tubular/scripts/update_recent_deployers.py]: I've added logging, so we'll be able to look and find out who's commit emails are being removed.

**Pull request merged**